### ISSUE TITLE
get following errors while I'm using the NERDTree in my vim

### ISSUE THREAD

**Issue by SBaof** [scrooloose/nerdtree#545]
```
Error detected while processing function nerdtree#ui_glue#invokeKeyMap..75..116..233..240:
line    3:
E117: Unknown function: strchars
E15: Invalid expression: strchars(a:line[0:numLeadBytes-1])
Press ENTER or type command to continue
```
OS version: CentOS
Vim version: 7.2 

Any answer is help! thx

**Comment by bhargavrpatel** [comment]
I get the same thing...

I am on Vim 7.2

**Comment by zhanghongzhou** [comment]
Same error
Any progress?

**Comment by zhanghongzhou** [comment]
Fixed. See issue#521 bottom

**Comment by PhilRunninger** [comment]
This is fixed in commit [4428aba](https://github.com/scrooloose/nerdtree/commit/4428aba2a299cbd117c80e7d18068b2cbb222fff) using @yous 's posted workaround in #517.
